lib/KafkaBacklogMetrics.js

```diff
@@ -69,9 +69,13 @@ class KafkaBacklogMetrics extends EventEmitter {
     }
 
     isReady() {
-        return this._zookeeper &&
+        if (!(this._zookeeper &&
             this._zookeeper.getState().code ===
-            zookeeper.State.SYNC_CONNECTED.code;
+            zookeeper.State.SYNC_CONNECTED.code)) {
+            this._log.error('KafkaBacklogMetrics is not ready: zookeeper not connected');
```

Comment:
Can we add the actual current code in the log, so that we know the current state of zookeeper?

Reply:
13ca620341c8dbd1847a882af78bb711ebde126a